web/impact/impact/v1/helpers/organization_helper.py

```diff
@@ -0,0 +1,67 @@
+from impact.models import Organization
+from impact.v1.helpers.model_helper import ModelHelper
+
+
+class OrganizationHelper(ModelHelper):
+    MODEL = Organization
+
+    REQUIRED_KEYS = [
+        "name",
+        "url_slug",
+        ]
+    OPTIONAL_KEYS = [
+        "additional_industry_ids",
+        "date_founded",
+        "facebook_url",
+        "full_elevator_pitch",
+        "primary_industry_id",
+        "public_inquiry_email",
+        "linked_in_url",
+        "location_city",
+        "location_national",
+        "location_postcode",
+        "location_regional",
+        "short_pitch",
+        "twitter_handle",
+        "video_elevator_pitch_url",
+        "website_url",
+        ]
+    INPUT_KEYS = REQUIRED_KEYS + OPTIONAL_KEYS
+    READ_ONLY_KEYS = [
+        "id",
+        "is_partner",
+        "is_startup",
+        "updated_at",
+        ]
+    OUTPUT_KEYS = READ_ONLY_KEYS + INPUT_KEYS
+
+    def __init__(self, *args, **kwargs):
+        super(OrganizationHelper, self).__init__(*args, **kwargs)
+        self.startup = self.subject.startup_set.order_by("-id").first()
+        self.partner = self.subject.partner_set.order_by("-id").first()
+
+    @property
+    def is_startup(self):
+        return self.startup is not None
+
+    @property
+    def is_partner(self):
+        return self.partner is not None
+
+    @property
+    def additional_industry_ids(self):
+        if self.startup:
```

Comment:
should we consider returning an empy list as a default?

Reply:
I could see this for Organizations that have Startups, but not for ones that only have Partners (since there really is no additional industry list.  However, given that it can be missing, I don't see the benefit of returning an empty list some of the time.  Won't fix unless I hear a clear argument why.